src/components/Bar_Main.js

```diff
@@ -10,10 +10,37 @@ export default class Bar_Main extends Component {
   }
 
   render() {
-    let { name } = this.props.cardData
+    let { name, imageURL, phone, notes, website, facebook, instagram, twitter } 
+      = this.props.cardData;
+    let { webIcon, phoneIcon, instaIcon, facebookIcon, twitterIcon}
```

Comment:
Love a good destructing! Great idea with the object of icons. I would like to check with Tiffany because I think she has a better eye, but we might want to try a single color for all social icons. Between the map, the colorful drag queen photos and the plethora of emojis in the bios, the cards already have a lot of color going on.